server/lib/statsd-collector.js

```diff
@@ -101,7 +109,7 @@ function sendMarketingImpressions(context, marketing, tags) {
 function sendNavigationTiming(context, navigationTiming, tags) {
   if (navigationTiming) {
     for (var key in navigationTiming) {
-      if (typeof navigationTiming[key] === 'number') {
+      if (isEventOffsetValid(navigationTiming[key], NAVIGATION_TIMING_MAX_OFFSET)) {
```

Comment:
I feel like filtering it is the right thing, because otherwise it's just going to skew the metrics.  But perhaps we should log it separately as some sort of error rather than dropping it entirely, so we don't lose a sense of how often it's happening in practice.